sdks/java/managed/src/main/java/org/apache/beam/sdk/managed/Managed.java

```diff
@@ -108,6 +110,7 @@ public class Managed {
    *
    * <ul>
    *   <li>{@link Managed#ICEBERG} : Read from Apache Iceberg tables
+   *   <li>{@link Managed#ICEBERG_CDC} : CDC Read from Apache Iceberg tables
```

Comment:
We should link to locations where users can find additional Javadocs related to each of these options (also for write).

Reply:
Added links to the connectors. 

Unfortunately only IcebergIO has fleshed-out documentation for Managed configuration parameters. Perhaps we should create a central location on the Beam website that displays these configuration options (similar to Dataflow: [iceberg](https://cloud.google.com/dataflow/docs/guides/managed-io-iceberg), [kafka](https://cloud.google.com/dataflow/docs/guides/managed-io-kafka), [bigquery](https://cloud.google.com/dataflow/docs/guides/managed-io-bigquery))